sharktank/tests/kernels/attention_template_test.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+import unittest
+from parameterized import parameterized
+
+import torch
+
+from iree.turbine import aot
+from sharktank import kernels
+from sharktank.types import layout_utils
+from sharktank.utils import debugging
+from sharktank import ops
+
+
+class custom_attention(unittest.TestCase):
+    def setUp(self):
+        torch.manual_seed(420)
+
+    @parameterized.expand(
+        [
+            (torch.float32, 5e-3, 1e-3, True),
+            (torch.float16, 5e-3, 1e-3, True),
+            # Currently failing on unmasked error
+            # (torch.float32, 5e-3, 1e-3, False),
+            # (torch.float16, 5e-3, 1e-3, False),
```

Comment:
I think you could xfail parametrized tests.
https://stackoverflow.com/questions/30571481/how-to-mark-test-as-xfail-only-with-specific-parameters-with-pytest